documentation/debugging.md

```diff
@@ -1,4 +1,88 @@
-# Debugging tests in Leviathan
+# Frequently Asked Questions (FAQ's)
+
+## Table of Contents
+  s- [Checking if Autokit is ready to run tests](#checking-if-autokit-is-ready-to-run-tests)
```

Comment:
nitpick: remove the `s` on the first line

Reply:
Done, the toc is autogenerated and it seems to have snuck in